notebooks/fs.ipynb

```diff
@@ -1565,9 +1565,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python [conda env:DEV-uwtools] *",
+   "display_name": "Python 3 (ipykernel)",
    "language": "python",
-   "name": "conda-env-DEV-uwtools-py"
```

Comment:
I think it's just metadata that's being pulled from the execution environment, whatever it might be. I just tested by building a package and creating a conda environment based on it -- so, independent of my devshell -- and everything appeared to work fine, though Jupyter again changed these values in the notebook code. I don't think it's worth fighting with Jupyter about its metadata: Unless we learn differently, I think we can just take what it gives us as long as the notebook functions correctly and their unit tests pass.